bin/export.py

```diff
@@ -122,9 +127,6 @@ def build_problem_zip(problem: Problem, output: Path):
     if problem.custom_output:
```

Comment:
(comment should actually be at lines 124–125, but that part is just outside the diff)

https://icpc.io/problem-package-format/spec/2023-07-draft.html#problem-statements says:

> Filetype can be either `.tex` for LaTeX files, `.md` for Markdown, or .pdf for PDF.

Should we remove `problem.pdf` from the root of the zip, and instead provide `statement/problem.<lang>.pdf`?

Same goes for `solution.<lang>.pdf`.

Reply:
I think I prefer that `bt pdf` (which has to be run from the top level directory) puts the files in the top level directory, but its fine for me to change this

Reply:
we might want to wait with this until we know how/if domjudge handles multi language support?

Reply:
Then still, I can imagine we build a kind of export where only the English version ends up in `statement/`, just like how we currently make sure only one language ends up in the zip as `problem.pdf` :slightly_smiling_face: But that's for later – I think that right now, we can strive to be as spec-compliant as possible, so other contest-system-implementors (primarily DOMjudge, but maybe also Kattis) can better test how their systems will handle them :smile:

Reply:
Turns out that the option to _not_ provide `.<lang>` has been dropped in https://github.com/Kattis/problem-package-format/commit/e42d7cadca0d18d1cda8fd51261a3d9c63d51f64 🙂 So I'd say we only write PDFs with language codes, for the new spec 😄 (I'd even say this if there was the option to drop the language code, because dropping the language code was optional :stuck_out_tongue:)

Reply:
I will have a look later today

Reply:
do we want to change the `problem.yaml` to also allow a dict instead of just a single name?

Reply:
For non-legacy export? Yes, I'd say we only need to reduce to a single name for the legacy export 🙂